nvtabular/io.py

```diff
@@ -380,6 +380,7 @@ def __init__(
         self.column_names = None
         if labels and conts:
             self.column_names = labels + conts
+        self.col_idx = {}
```

Comment:
If this is only used for HugeCTR it should be a member of the HugeCTRWriter class

Reply:
It is only used for HugeCTR, but only when usign Parquet format. So I can add it as a memeber of ParquetWriter Class.